packages/turf-standard-deviational-ellipse/index.js

```diff
@@ -12,7 +12,13 @@ import ellipse from '@turf/ellipse';
  * an ellipse that contains about one standard deviation’s worth (~ 70%) of the
  * data.
  *
- * This module was created in consultation with the following articles:
+ * This module mirrors the functionality of {@link
```

Comment:
🤔 ... I wouldn't split this up, maybe that's the cause of breaking?

If you are to create a link, keep it on the same line of code, not too fussy on a maximum character of lines.

Also I think we should be using Markdown to identify the links instead of `{@link`.

Back story for `{@link`: It's currently used as shortcut links using DocumentationJS, instead of inserting the hardcoded URL all the time, we can simple create a `{@link Point}` and the URL will be added from the following paths: https://github.com/Turfjs/turf/blob/master/scripts/generate-readmes#L22-L49

For all other links you can use Markdown to create links `[label](url)`